.circleci/config.yml

```diff
@@ -73,38 +67,6 @@ workflows:
               only: master
     jobs: *workflow_jobs
 jobs:
-  node4:
-    docker:
-      - image: 'node:4'
-        user: node
-    steps: &unit_tests_steps
```

Comment:
We have to move this section to the next-lowest version supported. This is how the nightly tests work, I believe.

@alexander-fenster do we want to make this change one-off here (in other words, do we want this PR), or can the generator do it across all of our projects at once?

Reply:
Both are fine (the generator generally refuses to make a change if it's already made, so nothing bad will happen if the node4 task is already deleted). But do we stop testing on node4 right now already? 

My understanding is that we want to wait at least for 4/30 when Node 4 is officially EOL. I'd like to add Node 10 into the same change so that we'd have one change across all repos, and not two.

In other words, I'd probably like to wait for 4/30 and then I'll make the change across all repos to disable node4 and add node10.

Reply:
And, for sure, removal of this section is wrong as it's referenced from other jobs.